services/harmony/app/util/cmr.ts

```diff
@@ -1091,3 +1092,30 @@ export function filterGranuleLinks(
   return granule.links.filter((g) => (g.rel.endsWith('/data#') || g.rel.endsWith('/service#'))
     && !g.inherited);
 }
+
+/**
+ * Returns CMR health information
+ *
+ * @returns A Promise with healthy and message indicating the CMR health info
+ */
+export async function isCmrHealthy(): Promise<{ healthy: boolean; message: string }> {
```

Comment:
nit: kind of weird to have a function name `isSomething` that doesn't return a boolean, or Promise<boolean>

Reply:
Renamed function.